jac-uk/admin
# new amendable panellist data w/permissions #2351

## What's included?

New fields for panellists;
- Correspondence Email
- Sex
- Ethnicity
*- Email *---changed-to-->* JAC Exercise Email

New fields viewable under `/panellists/view/[PANELLIST]`
Editable under edit tab
Edit tab only viewable for users with 'Can manage panellist data' permission. 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to Panellist section 
- View a Panellist 
- Check new fields are present and editable for users with permission
- Check new fields are uneditable for users without permission

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
🟠 Medium risk - this does change code that is shared with other areas
🔴 High risk - this includes a lot of changes to shared code

## Additional context
<img width="814" alt="image" src="https://github.com/jac-uk/admin/assets/44227249/d1162a76-77ec-41ec-b973-a9b7a99f55a6">

https://github.com/jac-uk/admin/assets/44227249/1b9b68e0-d4bb-462c-b58c-7959aab0a6a0


## Related permissions
Panellist > 'Can manage panellist data' permission necessary for editing data.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
